src/main/java/com/project/DemoApplication.java

```diff
@@ -11,8 +11,9 @@
 @ComponentScan({"com.project.controller"})
 @ComponentScan({"com.project.exception"})
 @ComponentScan({"com.project.validation"})
+@ComponentScan({"com.project.security"})
 public class DemoApplication {
-	public static void main(String[] args) {
-		SpringApplication.run(DemoApplication.class, args);
-	}
+    public static void main(String[] args) {
+        SpringApplication.run(DemoApplication.class, args);
```

Comment:
I highly doubt that your application is called `DemoApplication`.

Reply:
renamed DemoApplication class to ITAcademyBackend